app/routes/index.js

```diff
@@ -98,15 +119,15 @@ const showPostPage = (currentQuestion, request, h) => {
     (payload === {} || !Object.keys(payload).includes(yarKey) || payload[yarKey] === '')
   ) {
     const errorTextNoSelection = validate.errorEmptyField
-    return customiseErrorText(value, currentQuestion, errorList, errorTextNoSelection, yarKey, h)
+    return customiseErrorText(value, currentQuestion, errorList, errorTextNoSelection, yarKey, h, request)
   }
```

Comment:
do we have any way to decrease these arguments :))

Reply:
not really, because they are all needed.
you can check the function calls & parameters